packages/cerebral/package.json

```diff
@@ -18,7 +18,8 @@
   "homepage": "http://cerebral.github.io/cerebral-website",
   "dependencies": {
     "eventemitter3": "^2.0.2",
-    "function-tree": "^0.5.0-b"
+    "function-tree": "^0.5.0-b",
+    "prop-types": "^15.5.4"
```

Comment:
@christianalfoni  good catch. Maybe we could handle it in a similar way as here: https://github.com/paypal/glamorous/commit/14970c1bb59d1408d3066d209dafa356b5ce3173?  Try to require it only when React 15.5 detected and adding a note in the docs about `prop-types` installation?